hugo/README.md

```diff
@@ -0,0 +1,201 @@
+# NGINX Gateway Fabric Docs
```

Comment:
would it make sense to move this to https://github.com/nginxinc/nginx-gateway-fabric/tree/main/docs/developer ?

as a dev guide there - documentation.md or writing-documentation.md

writing docs has already been part of our workflow, so I think it make sense to keep it next to the other related files.

Additionally, I suggest add a section here about Generating Documentation -- https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/quickstart.md

Reply:
we can also link to that guide from https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/implementing-a-feature.md  Item 10 -- Update any relevant documentation:

Reply:
This makes sense. I have moved the file and relinked the files that have moved in the implementing-a-feature.md file.

I don't think we need to mention in-depth how to generate the documentation in the quickstar file. It's not required to get NGF running and the latest documentation will always be available at docs.nginx.com without the need to build it. I think it could be confusing for developers to have it there.